osquery_rules/mac/osquery_mac_enable_auto_update.py

```diff
@@ -0,0 +1,7 @@
+def rule(event):
```

Comment:
You probably want a separate alert per host? 
In which case I think the following would work:

```
def dedup(event):
   return event.get('hostIdentifier')
```

Reply:
Nah I don't want that, it would be super noisy